doc/src/bond_comm_epoxy_wire.md

```diff
@@ -346,17 +347,24 @@ connection for over 580 years before exhausting its conversation IDs. In
 practice, the `CONVERSATION_IDS_EXHAUSTED` error should never be
 encountered.
 
-### Method name ###
+### Service name ###
+
+The `serice_name` field is the name of the service that is being invoked.
+
+The `service_name` field must be set to a UTF-8 encoded string (without a
+BOM) that is the fully-qualified service name. Namespace elements are separated
+by the period character ('.', ASCII 0x2E). May be empty string if the message
```

Comment:
Minor: here and below, the phrasing "may be _the_ empty string" or "may be _an_ empty string" both read better to me.